qml/Keyboard.qml

```diff
@@ -272,7 +274,7 @@ function calculateSize()
     var newHeight;
     if( Settings.tabletUi ) {
         if( isLandscape ) {
-            newHeight = fullScreenItem.height * UI.tabletKeyboardHeightLandscapeM
+            newHeight = fullScreenItem.height * UI.tabletKeyboardHeightLandscapeL
```

Comment:
Yes when we're happy with caret stuff we can adapt that for language and keyboard size. Then we need to do some magic to actually resize it on selection.